### PR TITLE
hound CI and eslint configuration ##187300093 

### PR DESCRIPTION
**Pull request by manzidavidrw**
#### What does this PR do?

configure Hound CI and eslint

#### Description of Task to be completed?

- Configure The App to follow  rules
- Configure Hound to track issues

#### How should this be manually tested?

- to run lint use "npm run lint "
- Hound CI automatically runs it self whenever you push

#### Any background context you want to provide?

-this is for specifically allow order of codebase 

#### What are the relevant pivotal tracker/Trello stories?

- id of the project
  #187300093

### CODE REVIEW

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Hound not configured add it also

**Comment by manzidavidrw** [comment]
> Hound not configured add it also

hound check passed

**Review by Mukunzijames — APPROVED** [review]
Where is  eslint  configuration

File: .eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+          "jsx": true
+      }
+  },
+  "settings": {
+      "react": {
+          "version": "detect"
+      }
+  },
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
```
Comment (fab-ryan) [inline .eslintrc.json]: extension add tsx

File: .eslintrc.json [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+      "plugin:react/recommended",
+      "next/core-web-vitals"
+  ],
+  "plugins": ["react"],
+  "parserOptions": {
+      "ecmaVersion": 2020,
+      "sourceType": "module",
+      "ecmaFeatures": {
+          "jsx": true
+      }
```
Comment (fab-ryan) [inline .eslintrc.json]: here also add tsx axtensions

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Change that

**Review by Mukunzijames — APPROVED** [review]

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Please change target branch to be develop not main

**Comment by manzidavidrw** [comment]
> Please change target branch to be develop not main

Done

**Review by fab-ryan — APPROVED** [review]

**Review by Nadinefiona — APPROVED** [review]

**Pull request merged**